src/modules/StorageSnapper.rb

```diff
@@ -0,0 +1,151 @@
+# encoding: utf-8
+
+# Copyright (c) [2012-2015] Novell, Inc.
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+
+require "yast"
+
+
+module Yast
+
+  class StorageSnapperClass < Module
+
+
+    include Yast::Logger
+
+
+    def main
+
+      textdomain "storage"
+
+      Yast.import "String"
+      Yast.import "Installation"
+      Yast.import "InstExtensionImage"
+      Yast.import "Storage"
+
+    end
+
+
+    def BashLogOutput(command)
+      tmp = SCR.Execute(path(".target.bash_output"), command)
+
+      log.info("exit: #{tmp.fetch("exit")}")
+      tmp.fetch("stdout").each_line { |line| log.info("stdout #{line}") }
+      tmp.fetch("stderr").each_line { |line| log.info("stderr #{line}") }
+
+      return tmp.fetch("exit")
+    end
+
+
+    def ConfigureSnapper?()
+
+      part = Storage.GetEntryForMountpoint("/")
+
+      if part.fetch("used_fs", :unknown) != :btrfs
```

Comment:
I don't think the default ```:unknow``` is needed in this case. This would be more readable, IMO:
```ruby
return false unless part["used_fs"] == :btrfs
```